cowprotocol/cowswap
# Limit orders/receipt filled field

# Summary

This PR fixes the issue number 3 mentioned by @elena-zh in this comment https://github.com/cowprotocol/cowswap/pull/1502#pullrequestreview-1194347899
Basically it uses the same logic as it works on Explorer for calculating and displaying the filled amounts

## To test
- go to limit-orders page
- create few different orders
- open the receipt modal
- open the explorer link in modal
- make sure the values are same in receipt as they are in explorer
